mne_bids/meg_bids.py

```diff
@@ -90,6 +90,8 @@ def _events_tsv(events, raw, fname, event_id, verbose):
     df = pd.DataFrame(events[:, [0, 2]],
                       columns=['Onset', 'Condition'])
     if event_id:
+        if isinstance(event_id, str):
```

Comment:
you should be using string_types

`from mne.externals.six import string_types`

Reply:
I'm fixing this over in #28